src/components/variant/ui/change-variant-button/change-variant-button.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from './change-variant-button.module.css'
+
+import { ButtonHTMLAttributes, ReactElement } from 'react'
+import cn from 'classnames'
+
+import { Icon } from '@ui/icon'
+
+interface IChangeVariantButtonProps
+  extends ButtonHTMLAttributes<HTMLButtonElement> {
+  direction: 'up' | 'down'
+}
+
+export const ChangeVariantButton = ({
```

Comment:
I would name it in more generic way. This button is potentially reusable.

Reply:
Ok